scripts/line_length.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+from typing import Optional
+from typing import Sequence
+from subprocess import call
+
+
+def main(argv: Optional[Sequence[str]] = None) -> int:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("filenames", nargs="*", help="Filenames to check")
+    args = parser.parse_args(argv)
+
+    for filename in args.filenames:
+        with open(filename) as fd:
+            for lineno, line in enumerate(fd):
+                if len(line) > 100:
```

Comment:
factor the 100 out as either a flag or a top-level variable since this is an important value.  (Maybe someone else will want to copy this script but use a different value!)